### PR TITLE
[12.0][IMP] connector_jira: forced import option

### PR DESCRIPTION
**Pull request by alexey-pelykh**
@guewen @simahawk this PR adds option to force-sync already-imported items (via UI) by adding a checkbox

### CODE REVIEW

**Review by artemsenko — APPROVED** [review]
Functional tests

File: connector_jira/models/jira_backend/common.py [hunk 1]
```diff
@@ -295,7 +297,7 @@ def _inverse_delete_analytic_line_from_date(self):
         )
 
     @api.multi
-    def _run_background_from_date(self, model, from_date_field,
+    def _run_background_from_date(self, model, from_date_field, force,
```
Comment (simahawk) [inline connector_jira/models/jira_backend/common.py]: not a blocker but I'd keep it as kw arg here

**Review by simahawk — APPROVED** [review]

File: connector_jira/models/jira_backend/common.py (hunk 1 above)
Comment (alexey-pelykh) [reply connector_jira/models/jira_backend/common.py]: sorry, didn't get you. Add `kwargs` or move `force` to kwargs?

Comment (simahawk) [reply connector_jira/models/jira_backend/common.py]: the second

Comment (alexey-pelykh) [reply connector_jira/models/jira_backend/common.py]: sounds reasonable

**Pull request merged**